examples/python/invoke-rest-api/README.MD

```diff
@@ -0,0 +1,135 @@
+## About
+
+This function aims to provide a easy way to make a HTTP Post request to an API Endpoint that might help with a lot of integration scenarios. Eg. Post to a Slack channel, Create a PagerDuty or ServiceNow incident
+
+### Get the example function
+
+Clone this repository which contains the example functions. 
+
+```bash
+git clone https://github.com/vmware-samples/vcenter-event-broker-appliance
+cd vcenter-event-broker-appliance/examples/python/invoke-rest-api
+git checkout master
+```
+
+### Customize the function
+
+There are three key files, that you might have to modify if you are looking to customize this function and make a post api call to an external system. 
+
+```bash
+  /invoke-rest-api/metaconfig.json #sample copies are provided for pagerduty, slack and servicenow
+  /invoke-rest-api/stack.yml 
+  /invoke-rest-api/handler/handler.py
+```
+
+First, change the configuration file [metaconfig.json](metaconfig.json) holding both sensitive and configuration information needed for this function in this folder:
+
+The metaconfig (as you can see below) has inputs needed to make a POST API request such as - `url, auth, headers` and `body`. The `mapping` information provides a way to pull data from the Event that vCenter sends and replaces that within the Request body before making the POST call. 
+
+#### Example
+`"pull": "data/FullFormattedMessage"` attempts to get the value of `FullFormattedMessage` within the `data` dict of the cloud Event that we receive from VMware Event Router
+`"push": "payload/summary"` updates the `summary` field within the `payload` key of the request body as provided in this configuration file
+
+
+```json
+{
+  "url": "https://events.pagerduty.com/v2/enqueue",
+  "auth": {....},
+  "headers": {
+    "Authorization":"Bearer <TOKEN>",
+    .....
+  },
+  "body": {
+    "payload": {
+      "summary": "Example alert on host1.example.com",
+      .....
+    }
+  },
+  "mappings": [
+    {
+      "push": "payload/summary",
+      "pull": "data/FullFormattedMessage"
+    }
+    .....
+  ]
+}
+```
+
+Now let's store this configuration file as secret in the appliance.
+
+```bash
+# set up faas-cli for first use
+export OPENFAAS_URL=https://VEBA_FQDN_OR_IP
+faas-cli login -p VEBA_OPENFAAS_PASSWORD --tls-no-verify # vCenter Event Broker Appliance is configured with authentication, pass in the password used during the vCenter Event Broker Appliance deployment process
+
+# now create the secret
+faas-cli secret create metaconfig --from-file=metaconfig-pduty.json --tls-no-verify
+```
+
+> **Note:** Delete the local `metaconfig.json` after you're done with this exercise to not expose any sensitive information.
```

Comment:
I see inconsistent use of the `.json` file names across the readme. Can you please standardize on something like `metaconfig-[REST-API].json` so it matches the syntax of your examples provided?

Reply:
good idea, i'll use metaconfig-[SYSTEM].json